gulpfile.js

```diff
@@ -55,6 +55,23 @@ function styles(done) {
 }
 
 
+// I don't love this... I'd rather be able to tap into the core stylesheets
+gulp.task('gutenbergAdminStyles', gutenbergAdminStyles);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['gutenbergAdminStyles' was used before it was defined. (no-use-before-define)](https://app.codacy.com/app/uriweb/uri-component-library/pullRequest?prid=4632651)